package.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Is this top-level package.json and package-lock files required? Why wouldn't this package be installed under the `site` directory?